### ISSUE TITLE
request: pass column labels explicitly to biplot

### ISSUE THREAD

**Issue by jcpeterson** [erdogant/pca#7]
such as when a numpy matrix instead of pandas dataframe is given

**Comment by erdogant** [comment]
Thank you for your input! Can you give an example where/how the DataFrame is given as an input?

**Comment by jcpeterson** [comment]
Maybe I just haven't gotten the usage down. When I passed a numpy matrix, projections of the original dimensions had no labels, and I wasn't able to figure out how to supply them. Passing in my matrix as a dataframe worked, since it has column names.

**Comment by erdogant** [comment]
Below is an example with a Dataframe as input and separately Numpy array wit labels as input. 
Is this what you mean?

from pca import pca

```
# Initialize
model = pca(alpha=0.05, n_std=2)

# Example with Numpy array
X = np.array(np.random.normal(0, 1, 500)).reshape(100, 5)
row_labels = np.arange(0, X.shape[0]).astype(str)
# Fit transform
out = model.fit_transform(X, row_labels=row_labels)
# Make plot
model.biplot(legend=False)

# Example with DataFrame
X = pd.DataFrame(data=X, columns=np.arange(0, X.shape[1]).astype(str))
# Fit transform
out = model.fit_transform(X)
# Make plot
model.biplot(legend=False)
```

**Comment by jcpeterson** [comment]
Hmm ok, I see, thanks
